lib/core/macros.nim

```diff
@@ -890,6 +890,30 @@ proc boolVal*(n: NimNode): bool {.compileTime, noSideEffect.} =
   if n.kind == nnkIntLit: n.intVal != 0
   else: n == bindSym"true" # hacky solution for now
 
+macro expandMacros*(body: stmt): untyped =
```

Comment:
For some reason, `untyped` does not work. I was using statement because I though that typed would only work for a single expression, but it turns out it seems to work with a list of statements as well. I am pushing a version that uses `typed`